lib/components/ChangeWalletNetworkNotificationBanner.jsx

```diff
@@ -49,7 +48,7 @@ const ChangeWalletNetworkNotification = (props) => {
     <div className='flex flex-col sm:flex-row justify-between items-center'>
       <span>
         👋 Your wallet is currently set to <b>{walletChainName}</b>. Please connect to{' '}
-        <b>{poolChainName}</b> to participate in this pool.
+        <b>{poolChainName}</b> to participate.
```

Comment:
I liked the qualifier "in this pool" - if we leave it out, it sounds like you have to switch to that network to participate at all. (ie. please switch to Matic to participate)

Reply:
This notification shows up on the index page if you go to something like `/pools/matic` so having "in this pool" didn't quite make sense. And making this component award of context seemed more difficult than it was worth it.